function-reference/query-functions/text.xml

```diff
@@ -102,10 +102,12 @@ query variables and similar state.
   </tr>
   <tr>
     <ti>
-      <c>has_version pkg</c>
+      <c>has_version pkg[flag]</c>
     </ti>
     <ti>
-      True if <c>pkg</c> (can include version specifiers) is installed.
+      True if <c>pkg</c> (can include version specifiers) is installed. The use of <c>flag</c> is
+      optional and <c>flag</c> can be a list of multiple USE flags. Example:
```

Comment:
hmm, now that I think about it again, the proper name for those is [built with use dependencies](https://devmanual.gentoo.org/general-concepts/dependencies/index.html#built-with-use-dependencies). Maybe we can say something like: `True if <c>pkg</c> (can include version specifiers and built with use dependencies) is installed. Example:` ....